enatega-multivendor-store/lib/services/async-storage.ts

```
@@ -1,7 +1,51 @@
 import AsyncStorage from "@react-native-async-storage/async-storage";
-import { EventEmitter } from "events";
 
-export const asyncStorageEmitter = new EventEmitter();
+// Simple event emitter using a callback registry
+class SimpleEventEmitter {
+  private listeners: { [key: string]: Function[] } = {};
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)